lang/en/docs/package-json.md

```diff
@@ -363,6 +363,21 @@ If your package only allows one version of a given dependency, and you'd like to
 
 Note that if your `package.json` contains `"flat": true` and other packages depend on yours (e.g. you are building a library rather than an application), those other packages will also need `"flat": true` in their `package.json` or be installed with `yarn install --flat` on the command line.
 
+### `resolutions` <a class="toc" id="toc-resolutions" href="#toc-resolutions"></a>
+
+
+```json
+{
+  "resolutions": {
+    "transitive-package-1": "0.0.29",
+    "transitive-package-2": "file:./local-forks/transitive-package-2",
+    "dependencies-package-1/transitive-package-3": "^2.1.1"
+  }
+}
+```
+
+Allows you to override a version of a particular nested dependency. See [the Selective Versions Resolutions RFC](https://github.com/yarnpkg/rfcs/blob/master/accepted/0000-selective-versions-resolutions.md) for the full spec.
```

Comment:
This link will be changed soon the be under /implemented folder once this PR is merged https://github.com/yarnpkg/rfcs/pull/85